aspnetcore/mvc/views/razor.md

```diff
@@ -15,74 +15,74 @@ uid: mvc/views/razor
 
 By [Taylor Mullen](https://twitter.com/ntaylormullen) and [Rick Anderson](https://twitter.com/RickAndMSFT)
```

Comment:
Should update to include yourself

Reply:
@NTaylorMullen Do you want to be left in 1st? ... I'll slide myself in as 2nd on this. If you want to more-or-less hand it off, then I'll go 1st and @Rick-Anderson 2nd.

Reply:
I can be last. You guys have done most of the work.